code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -84,22 +84,23 @@
 
 /mob/living/carbon/xenomorph/proc/handle_critical_health_updates(seconds_per_tick)
 	if(loc_weeds_type)
-		heal_wounds(XENO_RESTING_HEAL, seconds_per_tick)
+		heal_wounds(XENO_RESTING_HEAL, seconds_per_tick = seconds_per_tick) // healing in critical while on weeds ignores scaling
```

Comment:
uh, why this change

Reply:
to just absolutely make sure that it's very obvious what arg this is touching, preventing mistakes, if you don't want it i can remove it

Reply:
you can already see what its doing because its an arg, literally jut hover over and it will say.

Reply:
changed it to positional args as it seemed what was seemingly requested.